www/src/components/profile/EabCredentials.js

```diff
@@ -37,28 +38,16 @@ function EabCredential({ credential, last }) {
         <TableData>{credential.provider}/{credential.cluster}</TableData>
         <TableData>{moment(credential.insertedAt).format('lll')}</TableData>
       </TableRow>
-      <Modal
+      <Confirm
         open={confirm}
-        onClose={() => setConfirm(false)}
-      >
-        <ModalHeader onClose={() => setConfirm(false)}>
-          Delete this EAB Credentials
```

Comment:
Should be "Delete this EAB Credential" (singular)

Reply:
Covid brain. 🤦‍♂️

Reply:
lol np